testing/visual-regression/backstop-common.js

```diff
@@ -184,6 +184,10 @@ module.exports = function backstopCommon(baseUrl) {
         focusSelector: '.cads-search__button',
         selectors: ['.cads-search'],
       },
+      {
```

Comment:
Not sure if I should cover anything else here

Reply:
This is fine for now I think